<launch>
<include file="$(find realsense2_camera)/rs_camera.launch"/>
  <arg name="camera:=935322071831"/>
<include file="$(find realsense2_camera)/rs_camera.launch"/>
  <arg name="camera:=939622074287"/>
<node pkg="dual_extrinsic_calibration" type="dual_extrinsic_calibration_node" name="dual_extrinsic_calibration_node"  output="screen"/>
  <param name="camera_1_image" value="/935322071831/color/image_raw" type="string"/>
  <param name="camera_2_image" value="/939622074287/color/image_raw"/>
  <param name="camera_1_info" value="/935322071831/color/camera_info"/>
  <param name="camera_2_info" value="/939622074287/color/camera_info"/>
<node pkg="dual_extrinsic_calibration" type="tfPosePublisher" name="tf_pose_broadcaster"/>
  <param name="parent_optical_frame" value="935322071831_color_optical_frame"/>
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find dual_extrinsic_calibration)/config/realsense.rviz"/>
</launch> 
